projects/packages/jitm/.gitignore

```diff
@@ -1,3 +1,4 @@
+/build
```

Comment:
I think you will also need to update the `.gitattributes` file to include that directory in production builds, and remove the previous `src/js/jetpack-jitm.min.js` file.

As you can see, the dist directory is currently not included in production builds:
https://github.com/Automattic/jetpack/actions/runs/1501782261

Reply:
I updated this in 3c6c3569da66f16245818136982cf49f3019a8b6